tests/common/snappi/common_helpers.py

```diff
@@ -436,68 +444,112 @@ def config_wred(host_ans, kmin, kmax, pmax, profile=None):
             return False
 
         """ Ensure that Kmin is no larger than Kmax during the update """
-        if kmin > kmin_old:
-            host_ans.shell('sudo ecnconfig -p {} -gmax {}'.format(p, kmax))
-            host_ans.shell('sudo ecnconfig -p {} -gmin {}'.format(p, kmin))
 
+        gmax_cmd = 'sudo ecnconfig -p {} -gmax {}'
+        gmin_cmd = 'sudo ecnconfig -p {} -gmin {}'
+
+        if asic_value != 'None':
+            gmax_cmd = 'sudo ip netns exec %s ecnconfig -p {} -gmax {}' % asic_value
```

Comment:
For the gmax/gmin cmd strings, a suggestion would be to introduce a "namespace" variable to encapsulate `ip netns exec ...` whenever namespaces are needed, instead of duplicating most of the string content.

Reply:
I agree with @kenneth-arista . @selldinesh  Can you please make the changes?